ansible/roles/products/common/local-hand-launch/tasks/main.yml

```diff
@@ -24,22 +24,32 @@
     right_hand_picture: hand_lite.jpg
     left_hand_picture: hand_lite_left.jpg
     bimanual_hands_picture: hand_lite_bimanual.jpg
-  when: product=='hand_lite'
+  when: '"hand_lite" in product'
 
 - name: Set icon picture filenames for hand_extra_lite
   set_fact:
     right_hand_picture: hand_extra_lite.jpg
     left_hand_picture: hand_extra_lite_left.jpg
     bimanual_hands_picture: hand_extra_lite_bimanual.jpg
-  when: product=='hand_extra_lite'
+  when: '"hand_extra_lite" in product'
+
+- name: set product variable
+  set_fact:
+    hand_type_value: "hand_type:=hand_g"
+  when: '"hand_extra_lite" in product or "hand_lite" in product'
```

Comment:
I think it would make more sense to set back again the defaults to "". The product parameter is quite important to be set properly and is quite often used, I noticed people having issues with the oneliner, so let's better have this as an aware choice (up to you).

Reply:
Sure thing reverting